### PR TITLE
doc: Add Tracker edition information

### LINKED ISSUE #26: Information on GNTE
#### Summary

Add a HW description of the GNTE

#### Why do we need this ?

To be able to show the device design and functionality

#### What is already there? What do you see now?

A coming soon page on the tracker edition.

#### What is missing? What do you want to see?

Filling this page with relevant information about the device

#### How do you propose to document this?

By adding relevant docs and HW description.

#### Can you do this yourself and submit a Pull Request?

All

### PR DESCRIPTION
**Pull request by elsalahy**
#### Summary

Closes #26 

#### Changes

- Add HW description of the tracker edition
- Add FW description of the tracker edition

#### Notes for Reviewers

This is a rushed one, so it doesn't have to perfect, so please provide your feedback based on this.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [x] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/tracker-edition/software/te-firmware/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "Tracker Edition Firmware"
+description: ""
+weight: -1
+distributions: null
+---
+
+The {{% gnte %}} [firmware](https://github.com/TheThingsIndustries/generic-node-te) allows for building multiple applications using [Semtech LR1110](https://www.semtech.com/products/wireless-rf/lora-edge/lr1110) modem.
```
Comment (benolayinka) [inline doc/content/tracker-edition/software/te-firmware/_index.md:8]:
```suggestion
The {{% gnte %}} [firmware](https://github.com/TheThingsIndustries/generic-node-te) allows for building multiple applications using the [Semtech LR1110](https://www.semtech.com/products/wireless-rf/lora-edge/lr1110) modem.
```

File: doc/content/tracker-edition/software/te-firmware/_index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+The STM32L073 host MCU communicates with the LR1110 modem over SPI bus, and it is able to perform multiple functionalities such as Wi-Fi and GNSS scanning and transmit the information using LoRaWAN.
```
Comment (benolayinka) [inline doc/content/tracker-edition/software/te-firmware/_index.md:10]:
```suggestion
The STM32L073 host MCU communicates with the LR1110 modem over SPI bus, and it is able to perform functions such as Wi-Fi and GNSS scanning, and transmit the information using LoRaWAN.
```

File: doc/content/tracker-edition/software/te-firmware/_index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+A simple Tracker example app joins the LoRaWAN network and performs a clock sync, and then periodically performs a Wi-Fi and GNSS scans and streams the scanned results to the network.
```
Comment (benolayinka) [inline doc/content/tracker-edition/software/te-firmware/_index.md:12]:
```suggestion
A simple Tracker example app joins the LoRaWAN network and performs a clock sync, periodically performs Wi-Fi and GNSS scans, and streams the scanned results to the network.
```

File: doc/content/tracker-edition/software/te-firmware/_index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+The application server solves the streamed data location coordinates provides an accurate location.
```
Comment (benolayinka) [inline doc/content/tracker-edition/software/te-firmware/_index.md:14]:
```suggestion
The application server solves the streamed data location coordinates and provides an accurate location.
```

**Review by benolayinka — APPROVED** [review]

**Review by benolayinka — APPROVED** [review]

**Comment by elsalahy** [comment]
Thanks @benolayinka ❤️

**Pull request merged**